scripts/install/opt/seagate/cortx/motr/libexec/motr_cfg.sh

```diff
@@ -160,22 +160,33 @@ set_key_value() # ARG1 [KEY] ARG2 [VALUE] ARG3 [FILE]
 do_m0provision_action()
 {
     local CFG_LINE=""
+    local LVM_SIZE_MIN=0
     local LVM_SIZE=0
 
-    msg "Configuring host [`hostname -f`]"
+    if [ ! -x /usr/sbin/lvs ]; then
+        err "lvs command not available."
```

Comment:
Shouldnt there be an exit in this case ?

Reply:
This function checks and update other config parameters which don't need lvs command. 
so can't return from this place. 
however we can add "MD_DEVICES=($(lvs -o lv_path 2>/dev/null | grep "lv_raw_metadata" | grep srvnode  | sort -u))" 
this line under else part of it.